trace/exporterutil/examples_test.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package exporterutil_test
+
+import (
+	"fmt"
+
+	"go.opencensus.io/trace"
+	"go.opencensus.io/trace/exporterutil"
+	"golang.org/x/net/context"
+)
+
+type testExporter struct {
+	buf string
+}
+
+func (e *testExporter) ExportSpan(spanData *trace.SpanData) {
+	e.buf += spanData.Name + "\n"
+}
+
+func (e *testExporter) Flush() {
+	fmt.Print(e.buf)
+}
+
+func ExampleExcludingSpans() {
+	filtered := exporterutil.ExcludingSpans(new(testExporter), exporterutil.WithNamePrefix("exclude-"))
```

Comment:
```
package traceutil

func FilteredExporter(e trace.Exporter, ...)
```

I'd discourage to use optional arguments here. What's the reason for the optional argument given an option struct can easily represent all the options and easier to document.